astroquery/lco/core.py

```diff
@@ -0,0 +1,216 @@
+/api-token-auth/    POST        Obtain an api token for use with authenticated requests.
+/frames/            GET         Return a list of frames.
+/frames/{id}/       GET         Return a single frame.
+/frames/{id}/related/ GET       Return a list of frames related to this frame (calibration frames, catalogs, etc).
+/frames/{id}/headers/ GET       Return the headers for a single frame.
+/frames/zip/        POST        Returns a zip file containing all of the requested frames. Note this is not the preferred method for downloading files. Use the frame's url property instead.
+/profile/           GET         Returns information about the currently authenticated user.
+
+The service accepts the following keywords,
+
+"""
+
+import json
+import keyring
+import getpass
+import warnings
+from datetime import datetime
+
+import astropy.units as u
+import astropy.coordinates as coord
+import astropy.io.votable as votable
+from astropy.table import Table
+from astropy.io import fits
+from astropy import log
+
+from ..query import BaseQuery, QueryWithLogin
+from ..utils import commons, system_tools, prepend_docstr_noreturns, async_to_sync
+from . import conf
+
+@async_to_sync
+class LcoClass(QueryWithLogin):
+
+    # use the Configuration Items imported from __init__.py to set the URL,
+    # TIMEOUT, etc.
+    URL = conf.server
+    TIMEOUT = conf.timeout
+    FRAMES_URL = conf.frames
+    DTYPES = ['i','S39','S230','i','S19','S16','S20','S3','S4','f','S2','i']
+    DATA_NAMES = ['id','filename','url','RLEVEL','DATE_OBS','PROPID','OBJECT','SITEID','TELID','EXPTIME','FILTER','REQNUM']
+    TOKEN = None
+
+    def query_object_async(self, object_name, get_query_payload=False,
+                           cache=True, start=None, end=None):
+        """
+        This method is for services that can parse object names. Otherwise
+        use :meth:`astroquery.lco.LcoClass.query_region`.
+        Put a brief description of what the class does here.
+
+        Parameters
+        ----------
+        object_name : str
+            name of the identifier to query.
+        get_query_payload : bool, optional
+            This should default to False. When set to `True` the method
+            should return the HTTP request parameters as a dict.
+        start: str, optional
+            Default is `None`. When set this must be in iso E8601Dw.d datestamp format
+            YYYY-MM-DD HH:MM
+        end: str, optional
+            Default is `None`. When set this must be in iso E8601Dw.d datestamp format
+            YYYY-MM-DD HH:MM
+        Returns
+        -------
+        response : `requests.Response`
+            The HTTP response returned from the service.
+            All async methods should return the raw HTTP response.
+
+        Examples
+        --------
+        While this section is optional you may put in some examples that
+        show how to use the method. The examples are written similar to
+        standard doctests in python.
```

Comment:
just a reminder to remove this reminder text and replace it with something useful